src/browserLauncher.ts

```diff
@@ -43,10 +43,11 @@ export class BrowserProcess {
   }
 }
 
-export async function launchBrowser(): Promise<BrowserProcess> {
+export async function launchBrowser(
+  browserExecutablePath,
+  headless
```

Comment:
```suggestion
  headless: boolean
```

I don't know if this is valid TypeScript, but giving types of arguments and return value is what I think will catch the most mistakes when working with type systems like this. (I have some experience with mypy, none with TypeScript.)